Codeinwp/otter-blocks
# Fix Slider in Sections

Closes #1125.


### Summary


Add `width` property. Default is `600px`.
### Screenshots 

----

### Test instructions


1. USE A FULL SITE EDITING THEME
2. Add a Section
3. Add a Slider to it.
4. See if it's working and play with the `Width`.



---- 

### Checklist before the final review
